packages/components/src/components/contract-card/contract-card-items/vanilla-options-card-body.tsx

```diff
@@ -48,9 +48,7 @@ const VanillaOptionsCardBody: React.FC<TVanillaOptionsCardBodyProps> = ({
                         {entry_spot_display_value && <Money amount={entry_spot_display_value} />}
                     </ContractCardItem>
 
-                    <ContractCardItem header={STRIKE}>
-                        {barrier && <Money amount={barrier} />}
-                    </ContractCardItem>
+                    <ContractCardItem header={STRIKE}>{barrier && <Money amount={barrier} />}</ContractCardItem>
```

Comment:
This and its test file changes are unrelated to this PR. lets revert this file changes

Reply:
This is just prettier changes
Do you want me to revert this? @amina-deriv

Reply:
lets revert them